python/GafferUI/NodeGraph.py

```diff
@@ -285,9 +294,11 @@ def __keyPress( self, widget, event ) :
 		elif event.key == "Down" :
 			selection = self.scriptNode().selection()
 			if selection.size() :
-				needsModifiers = not isinstance( selection[0], ( Gaffer.Reference, Gaffer.Box ) )
-				haveModifiers = bool( event.modifiers & ( event.modifiers.Shift | event.modifiers.Control ) )
-				if needsModifiers == haveModifiers :
+				needsModifiers = not Gaffer.Metadata.value( selection[0], "nodeGraph:childrenViewable" )
+				if (
+					( needsModifiers and event.modifiers == event.modifiers.Shift | event.modifiers.Control ) or
```

Comment:
It is more demanding - before you had to have at least one of Ctrl or Shift and now you need both. I did want to add Alt, but then I remembered the talk of using that for selection expansion and I decided against...